Tasks/AzureContainerAppsV0/task.json

```diff
@@ -0,0 +1,170 @@
+        },
+        {
+            "name": "imageToDeploy",
+            "type": "string",
+            "label": "Docker image to deploy",
+            "required": false,
+            "helpMarkDown": "The custom name of the image that has already been pushed to ACR and will be deployed to the Container App by this task. Note: the image name should include the ACR server; e.g., <acr-name>.azurecr.io/<repo>:<tag>. If this argument is not provided, the value provided (or determined) for the 'imageToBuild' argument will be used."
+        },
+        {
+            "name": "containerAppName",
+            "type": "string",
+            "label": "Azure Container App name",
+            "required": false,
+            "helpMarkDown": "The name of the Azure Container App that will be created or updated. If not provided, this value will be in the form of ado-task-app-<build-id>-<build-number>."
+        },
+        {
+            "name": "resourceGroup",
+            "type": "string",
+            "label": "Azure resource group name",
+            "required": false,
+            "helpMarkDown": "The resource group that the Azure Container App will be created in. If not provided, this value will be in the form of <container-app-name>-rg."
+        },
+        {
+            "name": "containerAppEnvironment",
+            "type": "string",
+            "label": "Azure Container App environment",
+            "required": false,
+            "helpMarkDown": "The name of the Azure Container App environment to use with the application. If not provided, an existing environment in the resource group of the Container App will be used, otherwise, an environment will be created in the format of <container-app-name>-env."
+        },
+        {
+            "name": "runtimeStack",
+            "type": "string",
+            "label": "Application runtime stack",
+            "required": false,
+            "helpMarkDown": "The platform version stack that the application runs in when deployed to the Azure Container App. This should be provided in the form of <platform>:<version>. If not provided, this value is determined by Oryx based on the contents of the provided application. Please view the following document for more information on the supported runtime stacks for Oryx: https://github.com/microsoft/Oryx/blob/main/doc/supportedRuntimeVersions.md"
+        },
+        {
+            "name": "targetPort",
+            "type": "string",
+            "label": "Application target port",
+            "required": false,
+            "helpMarkDown": "The designated port for the application to run on. If not provided, this value is 80 for Python applications, and 8080 for all other supposed platforms."
+        }
+    ],
+    "execution": {
+        "Node10": {
+            "target": "azurecontainerapps.js"
```

Comment:
Currently, we are migrating our tasks to the node 16 handler, could you please also add it and test it?

Reply:
Followed the guidance from the `migrateNode16.md` -- feel free to let me know if there are any additional changes that should be made after the latest set of commits I push.